core/domain/suggestion_registry.py

```diff
@@ -403,10 +403,10 @@ def validate(self):
         if not (
                 question_state_schema_version >= 1 and
                 question_state_schema_version <=
-                feconf.CURRENT_EXPLORATION_STATES_SCHEMA_VERSION):
+                feconf.CURRENT_STATES_SCHEMA_VERSION):
             raise utils.ValidationError(
                 'Expected question state schema version to be between 1 and '
-                '%s' % feconf.CURRENT_EXPLORATION_STATES_SCHEMA_VERSION)
+                '%s' % feconf.CURRENTSTATES_SCHEMA_VERSION)
```

Comment:
Make sure to read through your diff immediately after submitting to GitHub (or, ideally, before submitting, using a tool like "meld").